.travis.yml

```diff
@@ -14,3 +14,11 @@ script:
     - sh build.sh
     # Invoke ci script too
     - sh contrib/ci.sh
+after_success:
+    - if [[ "$TRAVIS_TAG" != null ]]; then 
+        docker tag functions/gateway:latest-dev $DOCKER_USERNAME/gateway:latest;
```

Comment:
It may be OK to overwrite the tag for stable releases, but sometimes we do a pre-release build too. For these `latest` should not be overwritten.